network/robus/inc/config.h

```diff
@@ -45,8 +45,8 @@
 #endif
 
 // Tab of byte. + 2 for overlap ID because aligned to byte
-#define MASK_SIZE ((MAX_SERVICE_NUMBER / 8) + 2)
-
+#define MASK_SIZE           ((MAX_SERVICE_NUMBER / 8) + 2)
```

Comment:
ID_MASK_SIZE instead of MASK_SIZE

Reply:
@nicolas-rabault I agree that we should change the target_mode = ID to SERVICE but when we will do the major Luos clean and not now. If you want create an issue regarding this or I can do it

Reply:
Agree, yes please feel free to create an issue about it.